packages/web/src/features/users/LoginView.tsx

```diff
@@ -71,9 +72,19 @@ export default function LoginView() {
           />
         </div>
         <div className="mb-6">
-          <FormLabel htmlFor="password">
-            {t('users:password').toUpperCase()}
-          </FormLabel>
+          <div className="flex flex-row justify-between">
+            <FormLabel htmlFor="password">
+              {t('users:password').toUpperCase()}
+            </FormLabel>
+            <Button
+              small
+              variant="tertiary"
+              className="italic text-xs"
+              to={'/forgot-password'}
+            >
+              {t('users:forgot_password')}
+            </Button>
```

Comment:
I forgot to include this in the ticket, but I had originally designed the forgot password to be below the login button. That's a pretty common place it goes in login views

<img width="1001" alt="Screen Shot 2024-03-27 at 9 23 35 PM" src="https://github.com/globalbibletools/gbt/assets/11968925/8bbc382e-7bd0-4952-89f1-47d6877b7c25">

Reply:
Done. By the way, is there a place where our new figma designs are located? It seems we've long moved away from https://www.figma.com/file/GYKTJWl1otgwpmA9ZLLthE/Gloss-Translation-App?type=design&node-id=1023-276&mode=design&t=pYANiBhuPrIb150t-0